### DISCUSSION TITLE
Client Simulation problem

### DISCUSSION THREAD

**Discussion by CalvinKoung** [bitcraze/discussions#943] [Q&A] [answered]
Hi, I want to use UWB to do indoor navigation. So I want to know the UWB signals stability and measurement distance with crazyflie tag and anchor. In this plotter. Is the X axis is meter between tag and anchor ?  And what is the other options under plotter means ? thanks. 
![4BDFE659-C594-4FD3-BF60-B06F11977DBA](https://github.com/bitcraze/discussions/assets/118436188/da8139ee-30c7-4404-ae07-92cc4df470d1)

**Comment by krichardsson** [comment, marked as answer]
You can read about the log subsystem, the [plotter tab](https://www.bitcraze.io/documentation/repository/crazyflie-clients-python/master/userguides/userguide_client/plotter_tab/) and [how to configure your own logging configurations in the client documentation](https://www.bitcraze.io/documentation/repository/crazyflie-clients-python/master/userguides/userguide_client/#logging).

The log configuration you used in the screen shot is actually set up automatically by the Loco tab and is used to visualize the system, this is fine to use but you can also create your own configurations if you want to log some other data. Note that there is a limit to how many variables that can be added to a log configuration, if needed you must create multiple log configurations.

**Reply by CalvinKoung** [reply]
ok, thanks

**Reply by CalvinKoung** [reply]
@krichardsson Hi, Is there any way to open multiple windows for monitoring at one time?

**Reply by knmcguire** [reply]
each of the tabs you can also open as toolboxes, which enables you to put them next to each other:

https://www.bitcraze.io/documentation/repository/crazyflie-clients-python/master/userguides/userguide_client/#tabs-and-toolboxes

For future support, we prefer one issue per discussion, so just open up a new discussion if you have another question.

**Reply by CalvinKoung** [reply]
OK, I got it. Thanks.
